src/main/scala/edu/ie3/simona/config/ArgsParser.scala

```diff
@@ -177,6 +177,7 @@ object ArgsParser extends LazyLogging {
           ReflectionTools
             .resolveClassNameToCompanion(listenerElem.fullClassPath) match {
             case Some(listener: SimonaListenerCompanion) =>
+              // TODO: Double check if we should have an Optional here by design
```

Comment:
So you mean rather `resolveClassNameToCompanion` returning the `Try` and working with that here?

Reply:
This comment was from @t-ober. From my perspective, we can leave this as it is now.